tests/cephadm/test_package_validation.py

```diff
@@ -0,0 +1,159 @@
+import re
+
+from cli.utilities.containers import Container
+from cli.utilities.packages import Package, PackageError, ReposError, Rpm
+from cli.utilities.utils import get_custom_repo_url
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class PackageValidationError(Exception):
+    pass
+
+
+YUM_REPO_DIR = "/etc/yum.repos.d"
+
+installed_packages = [
+    "chrony",
+    "lvm2",
+    "podman",
+]
+
+packages_to_install = [
+    "cephadm-ansible",
+]
+
+rpm_dependent_packages = [
+    "ansible-core",
+    "ansible-collection-ansible-posix",
+    "ansible-collection-community-general",
+]
+
+rpm_client_packages = [
+    "ceph-common",
+    "ceph-fuse",
+    "cephfs-top",
+]
+
+
+def validate_rpm_version(node, package_list, version_config):
+    """Validate installed rpm package version
+
+    Args:
+        node (ceph.ceph.CephNode): Node name
+        package_list (list): List of packages
+        version_config (dict): List of packages and package version
+    """
+    # Define regex to get version
+    rpm_regex = r"Version\s+:\s+(\d+\.\d+)"
+
+    # Check rpm package version using yum info
+    for pkg in package_list:
+        out = Package(node).info(pkg)
+        version = re.search(rpm_regex, out).group(1)
+        if version < version_config.get(pkg):
+            raise PackageError(
+                f"Expected version for {pkg} is {version_config.get(pkg)}; found {version}"
+            )
+        else:
+            log.info(f"{pkg} installed correctly with version {version}")
+
+
+def validate_image_version(node, image_dict, version_config):
```

Comment:
same as above

Reply:
@vamahaja do you mean I should it to lib?

Reply:
Same as above add here - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/containers.py#L47